dev/automation-hub/galaxy_ng_standalone.env

```diff
@@ -0,0 +1,9 @@
+PULP_CONTENT_PATH_PREFIX=/api/automation-hub/v3/artifacts/collections/
```

Comment:
I'd prefer using directory rather than a suffix to indicate a profile:

```
dev/common/galaxy_ng.env
dev/insights/galaxy_ng.env
dev/standalone/galaxy_ng.env
```